specification/synapse/data-plane/Microsoft.Synapse/stable/2020-12-01/entityTypes/Pipeline.json

```diff
@@ -3276,6 +3276,21 @@
         }
       }
     },
+    "GoogleBigQueryV2Source": {
+      "description": "A copy activity Google BigQuery service source.",
+      "type": "object",
+      "allOf": [
+        {
+          "$ref": "#/definitions/TabularSource"
+        }
+      ],
+      "properties": {
+        "query": {
+          "type": "object",
+          "description": "A query to retrieve data from source. Type: string (or Expression with resultType string)."
```

Comment:
why is this not a string and just an object of no specific type?

Reply:
In ADF and synapse, for the query, we support the dynamic expression the type is decided in backend service. So we set the type as object align with existing app model type.